data/fleet-commander-standalone.in

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ $UID != 0 ]; then
+    echo "WARNING: You need to be root in order to have access to dbus service"
```

Comment:
I am not sure this sentence is helpful at all for the user, the user might not even know what dbus is.

"WARNING: You need to be root in order to access the live session settings editor"

And in any case the app is going to be faulty (clicking in settings won't work and won't let the user know what is going on).

I'm tempted to say we should just fail and ask the user to run this with sudo